sklearn/metrics/classification.py

```diff
@@ -313,8 +315,11 @@ class labels [2].
     .. [1] J. Cohen (1960). "A coefficient of agreement for nominal scales".
            Educational and Psychological Measurement 20(1):37-46.
            doi:10.1177/001316446002000104.
-    .. [2] R. Artstein and M. Poesio (2008). "Inter-coder agreement for
+    .. [2] `R. Artstein and M. Poesio (2008). "Inter-coder agreement for
```

Comment:
Aside: It looks like [t-sne](https://github.com/hlin117/scikit-learn/blob/master/sklearn/manifold/t_sne.py#L634) doesn't make reference links at all. Do you want me to address this in this PR as well?